ArduCopter/autoyaw.cpp

```diff
@@ -123,12 +123,10 @@ void Mode::AutoYaw::set_fixed_yaw(float angle_deg, float turn_rate_dps, int8_t d
         // default to regular auto slew rate
         _fixed_yaw_slewrate = AUTO_YAW_SLEW_RATE;
     } else {
-        const int32_t turn_rate = (wrap_180_cd(_fixed_yaw - curr_yaw_target) / 100) / turn_rate_dps;
-        _fixed_yaw_slewrate = constrain_int32(turn_rate, 1, 360);    // deg / sec
+        _fixed_yaw_slewrate = wrap_360_cd(turn_rate_dps) * 100;
```

Comment:
@tajisoft san.  Yaw rotation can be specified as relative or absolute angle.
In the case of a relative angle, the parameter's angular velocity seems to be sufficient.
However, for absolute angles, the specified angular rate will not be used.
For example, if the angle is currently 90 degrees and the absolute angle is 180 degrees, the movement angle will be 90 degrees.
The planner does not need to know the current angle to make the angle you specify.
I think this PR requires the planner to know the current angle.

Reply:
@muramura Thank you so much for your advise. And I'm sorry about the delay on getting back to this. I try to understand your point.